pkg/store/store-api_test.go

```diff
@@ -286,9 +287,7 @@ func Test_device1_version(t *testing.T) {
 		fmt.Printf("%d: %s\n", idx, B64([]byte(cid)))
 	}
 
-	if device1V.Name != "Device1Version" {
-		t.Errorf("Unexpected name for Configuration main %s", device1V.Name)
-	}
+	assert.Assert(t, device1V.Name == "Device1Version", "Unexpected name for Configuration main %s", device1V.Name)
```

Comment:
device1V.Name is a ConfigName which is an alias for string
"Device1Version" is a string 
Use 
assert.Equal(t, device1V.Name == ConfigName("Device1Version"))